paddle/fluid/pybind/pybind.cc

```diff
@@ -181,6 +184,53 @@ bool SupportsBfloat16FastPerformance() {
 #endif
 }
 
+std::tuple<std::unordered_set<std::string>, std::unordered_set<std::string>,
```

Comment:
Something for the readability, please add simple comment.

The function checks all ops supports input place and dtype, returns all_ops, supported_ops, and unsupported_ops. The function name and return type are not obvious enough for above information.

Reply:
Done.